RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py

```diff
@@ -22,10 +22,11 @@
 )
 # AOD content
 RecoLocalMuonAOD = cms.PSet(
-    outputCommands = cms.untracked.vstring(
+    outputCommands = cms.untracked.vstring('keep *_dt1DRecHits_*_*', 
         'keep *_dt4DSegments_*_*', 
         'keep *_dt4DCosmicSegments_*_*',
         'keep *_cscSegments_*_*', 
+        'keep *_csc2DRecHits_*_*', 
```

Comment:
for 10_6_X (this PR) this part should be done below for `bParking` modifier
```
_updateOutput(bParking, RecoLocalMuonAOD, ['keep *_dt1DRecHits_*_*', 'keep *_csc2DRecHits_*_*'])
```

Reply:
BTW, was it confirmed that `dt1DRecHits` work OK, or should this be `dt1DCosmicRecHits` instead?

Reply:
We checked with the analysis team who uses the DT rechits, we do not need `dt1DCosmicRecHits`

Reply:
@slava77 Done in 7632eaff176b865807c4974439692cd5e757ca19